### ISSUE TITLE
a11y_Radio> Default> A11y> 1 violations and 1 incomplete is appearing

### ISSUE THREAD

**Issue by priya-rajput** [cengage/react-magma#514]
**Describe the bug**
Radio>> "1 violations and 1 incomplete" is appearing for "Default". 

**To Reproduce**
1. Go to 'https://storybook-preview-290--upbeat-sinoussi-f675aa.netlify.app/?path=/story/radio--default'
2. Click on 'Accessibility" add on
3. "1 violations and 1 incomplete" is appearing

**Expected behavior**
"0 violations and 0 incomplete" should appear for Radio> Default.

**Screenshots**
https://somup.com/c3n2QsTccV

**Desktop:**
 - OS: [Win 10]
 - Browser [chrome]
 - Version [Version 98.0.4758.82 (Official Build) (64-bit)]

**Comment by Uditsingh67** [comment]
The issue is fixed.
Storybook Link : https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/radio--default
See Screen-cast : http://somup.com/c3jQQNuJdz
Thanks!

**Comment by Uditsingh67** [comment]
Issue does not exist.
See Screen-cast : http://somup.com/c3jwirU2TQ
